braindecode/augmentation/_functionals.py

```diff
@@ -0,0 +1,746 @@
+""" This module contains the internal functions defining data augmentations.
+They are not supposed to be part of the public API.
+"""
+
+# Authors: Cédric Rommel <cpe.rommel@gmail.com>
+#
+# License: BSD (3-clause)
+
+from numbers import Real
+
+import numpy as np
+from scipy.interpolate import Rbf
+from sklearn.utils import check_random_state
+import torch
+from torch.fft import fft, ifft
+from torch.nn.functional import pad, one_hot
+from mne.filter import notch_filter
+from mne.channels import make_standard_montage
+
+
+def identity(x, y, *args, **kwargs):
```

Comment:
is it necessary to have these *args, **kwargs in all of these functions in the signature?

Reply:
No, it was for compatibility with another repo of ours, but it should not be in braindecode as discussed with @agramfort. I'm going to remove them 👍